Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XLGPlusPlus/BXLInvocationAnalyzer.cs

```diff
@@ -78,10 +85,16 @@ internal sealed class BXLInvocationAnalyzer : Analyzer
         public override int Analyze()
         {
             var dataStore = new XldbDataStore(storeDirectory: InputDirPath);
-
-            File.WriteAllLines(OutputFilePath, dataStore.GetEventsByType(Xldb.ExecutionEventId.DominoInvocation));
+            File.AppendAllLines(OutputFilePath, dataStore.GetBXLInvocationEvents());
 
             return 0;
         }
+
+        /// <inheritdoc/>
+        protected override bool ReadEvents()
+        {
+            // Do nothing. This analyzer does not read events.

```

Comment:
>/ Do nothing. [](start = 13, length = 13)

so why do you need this method then?

Reply:
Because otherwise, it would invoke "Input.ReadExecutionLog" which we do not want to do anymore.

---
In reply to: [309831355](https://github.com/microsoft/BuildXL/pull/685#discussion_r309831355) [](ancestors = 309831355)